test/services/roster/index.test.js

```diff
@@ -46,73 +45,74 @@ describe('roster service', function oauth() {
+			pseudonymService
+				.find({
+					query: {
+						userId: testUser1._id,
+						toolId: testTool1._id,
+					},
+				})
+				.then((pseudonym) => {
+					pseudonym1 = pseudonym.data[0].pseudonym;
+					return Promise.resolve();
+				})
+		);
 	});
 
-	after(() => Promise.all([
-		pseudonymService.remove(null, { query: {} }),
-		toolService.remove(testTool1),
-		courseService.remove(testCourse._id),
-	]).then(server.close()));
+	after(() =>
+		Promise.all([
+			pseudonymService.remove(null, { query: {} }),
+			toolService.remove(testTool1),
+			courseService.remove(testCourse._id),
+		]).then(server.close())
+	);
 
 	it('is registered', () => {
 		assert.ok(metadataService);
 		assert.ok(userGroupsService);
 		assert.ok(groupsService);
 	});
 
-	it('GET metadata', () => metadataService
-		.find({ route: { user: pseudonym1 } })
-		.then((metadata) => {
+	it('GET metadata', () =>
+		metadataService.find({ route: { user: pseudonym1 } }).then((metadata) => {
 			assert.strictEqual(pseudonym1, metadata.data.user_id);
 			assert.strictEqual('teacher', metadata.data.type);
 		}));
 
 	it('GET user groups', (done) => {
-		userGroupsService.find({
-			route: { user: pseudonym1 },
-			tokenInfo: { client_id: testTool1.oAuthClientId },
-		}).then((groups) => {
-			const group1 = groups.data.groups[0];
-			assert.strictEqual(testCourse._id, group1.group_id);
-			assert.strictEqual(testCourse.name, group1.name);
-			assert.strictEqual(testCourse.userIds.length, group1.student_count);
-			done();
-		});
+		userGroupsService
+			.find({
+				route: { user: pseudonym1 },
+				tokenInfo: { client_id: testTool1.oAuthClientId },
+			})
+			.then((groups) => {
+				const group1 = groups.data.groups[0];
+				assert.strictEqual(testCourse._id, group1.group_id);
+				assert.strictEqual(testCourse.name, group1.name);
+				assert.strictEqual(testCourse.userIds.length, group1.student_count);
+				done();
+			});
 	});
 
 	it('GET group', (done) => {
-		groupsService.get(testCourse._id, {
-			tokenInfo: {
-				client_id: testTool1.oAuthClientId,
-				obfuscated_subject: pseudonym1,
-			},
-		}).then((group) => {
-			assert.strictEqual(pseudonym1, group.data.teachers[0].user_id);
-			const properties = 'title="username" style="height: 26px; width: 180px; border: none;"';
-			const iframe = `<iframe src="http://localhost:3100/oauth2/username/${pseudonym1}" ${properties}></iframe>`;
-			assert.strictEqual(encodeURI(iframe), group.data.teachers[0].username);
-			done();
-		});
+		groupsService
+			.get(testCourse._id, {
+				tokenInfo: {
+					client_id: testTool1.oAuthClientId,
+					obfuscated_subject: pseudonym1,
+				},
+			})
+			.then((group) => {
+				assert.strictEqual(pseudonym1, group.data.teachers[0].user_id);
+				const properties = 'title="username" style="height: 26px; width: 180px; border: none;"';
+				const iframe = `<iframe src="http://localhost:3100/oauth2/username/${pseudonym1}" ${properties}></iframe>`;
```

Comment:
Codacy found an issue: [This line has a length of 123. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)